go-peer/main.go

```diff
@@ -169,7 +170,10 @@ func main() {
 		libp2p.Identity(privk),
 		libp2p.Transport(quicTransport.NewTransport),
 		libp2p.Transport(webtransport.New),
+		libp2p.Transport(webrtc.New),
 		libp2p.ListenAddrStrings("/ip4/0.0.0.0/udp/9095/quic-v1", "/ip4/0.0.0.0/udp/9095/quic-v1/webtransport"),
+		// 👇 webrtc-direct cannot listen on the same port as QUIC or WebTransport
+		libp2p.ListenAddrStrings("/ip4/0.0.0.0/udp/9096/webrtc-direct"),
```

Comment:
In the latest version of go-libp2p, the same port can be reused for WebRTC, QUIC, and WebTransport, and even TCP. 
```suggestion
		libp2p.ListenAddrStrings("/ip4/0.0.0.0/udp/9095/webrtc-direct"),
```

Reply:
And let's remove the comment as well